groovy/inductor/inductor_xpu_backend_e2e.groovy

```diff
@@ -0,0 +1,304 @@
+node(env.nodes_label){
+    cleanWs()
+    stage('setup env') {
+        println('================================================================')
+        println('setup nodes env')
+        println('================================================================')
+        sh '''
+        set -e
+        set +x
+        if [ -e ${HOME}/miniconda3/etc/profile.d/conda.sh ];then
+            source ${HOME}/miniconda3/etc/profile.d/conda.sh 2>&1 >> /dev/null
+            if [ -d ${HOME}/miniconda3/envs/${conda_env} ];then
+                conda activate ${conda_env}
+            else
+                conda create -n ${conda_env} cmake ninja pillow python=${python_version}
+            fi
+        else
+            wget -q -e "https_proxy=http://proxy-us.intel.com:912" https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+            bash Miniconda3-latest-Linux-x86_64.sh -b
```

Comment:
Do you have met the miniconda package issue like https://github.com/conda/conda/issues/13225 recently? Seems the package https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh (same as [Miniconda3-py311_23.9.0-0-Linux-x86_64.sh](https://repo.anaconda.com/miniconda/Miniconda3-py311_23.9.0-0-Linux-x86_64.sh)) has issue now. suggest to use other python versions, eg. [Miniconda3-py39_23.9.0-0-Linux-x86_64.sh](https://repo.anaconda.com/miniconda/Miniconda3-py39_23.9.0-0-Linux-x86_64.sh)

Reply:
Never met this issue. We usually install the Miniconda3-latest-Linux-x86_64.sh on our machine firstly

Reply:
Ok, please aware it, when we meet such issue can use similar workaround